sdk/core/Azure.Core/src/Shared/ClientDiagnostics.cs

```diff
@@ -65,154 +60,6 @@ internal static HttpMessageSanitizer CreateMessageSanitizer(DiagnosticsOptions d
-                // included verbatim in the detailed error message
-            }
-
-            return null;
-        }
-
-        public async ValueTask<RequestFailedException> CreateRequestFailedExceptionAsync(Response response, ResponseError? error = null, IDictionary<string, string>? additionalInfo = null, Exception? innerException = null)
-        {
-            if (GetType() == typeof(ClientDiagnostics) && error is null && additionalInfo is null)
-            {
-                return new RequestFailedException(response, innerException);
-            }
-
-            var content = await ReadContentAsync(response, true).ConfigureAwait(false);
-            return CreateRequestFailedExceptionWithContent(response, error, content, additionalInfo, innerException);
-        }
-
-        public RequestFailedException CreateRequestFailedException(Response response, ResponseError? error = null, IDictionary<string, string>? additionalInfo = null, Exception? innerException = null)
-        {
-            if (GetType() == typeof(ClientDiagnostics) && error is null && additionalInfo is null)
-            {
-                return new RequestFailedException(response, innerException);
-            }
-
-            string? content = ReadContentAsync(response, false).EnsureCompleted();
-            return CreateRequestFailedExceptionWithContent(response, error, content, additionalInfo, innerException);
-        }
-
-        private RequestFailedException CreateRequestFailedExceptionWithContent(
-            Response response,
-            ResponseError? error = null,
-            string? content = null,
-            IDictionary<string, string>? additionalInfo = null,
-            Exception? innerException = null)
-        {
-            error ??= ExtractAzureErrorContent(content);
-            var formatMessage = CreateRequestFailedMessageWithContent(response, error, content, additionalInfo, _sanitizer);
-            var exception = new RequestFailedException(response.Status, formatMessage, error?.Code, innerException);
-
-            if (additionalInfo != null)
-            {
-                foreach (KeyValuePair<string, string> keyValuePair in additionalInfo)
-                {
-                    exception.Data.Add(keyValuePair.Key, keyValuePair.Value);
-                }
-            }
-
-            return exception;
-        }
-
-        public async ValueTask<string> CreateRequestFailedMessageAsync(Response response, ResponseError? error, IDictionary<string, string>? additionalInfo, bool async)
-        {
-            var content = await ReadContentAsync(response, async).ConfigureAwait(false);
-            return CreateRequestFailedMessageWithContent(response, error, content, additionalInfo, _sanitizer);
-        }
-
-        internal static string CreateRequestFailedMessageWithContent(Response response, ResponseError? error, string? content, IDictionary<string, string>? additionalInfo, HttpMessageSanitizer sanitizer)
```

Comment:
Where did all this formatting move to then? I haven't seen it yet or from a cursory glance, or was it always duplicated in RFE?

Reply:
It was already duplicated.